src/core/ext/filters/client_channel/resolver/xds/xds_resolver.cc

```diff
@@ -296,6 +239,63 @@ class XdsResolver : public Resolver {
     RefCountedPtr<XdsResolver> resolver_;
   };
 
+  // An entry in the map of clusters that need to be present in the LB
+  // policy config.  The map holds a weak ref.  One strong ref is held by
+  // the ConfigSelector, and another is held by each call assigned to
+  // the cluster by the ConfigSelector.  The ref for each call is held
+  // until the call is committed.  When the strong refs go away, we hop
+  // back into the WorkSerializer to remove the entry from the map.
+  class ClusterState : public DualRefCounted<ClusterState> {
+   public:
+    using ClusterStateMap =
```

Comment:
This typedef is no longer needed as of #32989.

Reply:
Done